plugin-enclave-gradle/src/main/resources/com/r3/conclave/plugin/enclave/gradle/gramine/java.manifest.template

```diff
@@ -23,11 +23,11 @@ sgx.isvsvn    = {{ isv_svn }}
 {% if is_python_enclave == "true" %}
 loader.env.LD_LIBRARY_PATH = "/lib:{{ arch_libdir }}:/usr{{ arch_libdir }}:/usr/lib:{{ python_packages_path }}/jep"
 loader.env.PYTHONPATH = "{{ python_packages_path }}"
+loader.env.LD_PRELOAD = "{{ python_ld_preload }}"
```

Comment:
I would keep the arg just `ld_preload`. We might need it even without Python.

Reply:
I am ok to rename it to `ld_preload`, but I would keep this still inside the Python presence condition.

I was bitten by an error then I felt it is clearer to change it/use it only when actually needed.

My understanding is also that we currently rely on Python being installed in order to feed this `LD_PRELOAD` parameter that is not useful for non-Python enclaves. This is somehow not nice/incorrect.
But, more importantly, providing an empty value would cause issues when Gramine generates the manifest.
So I felt that moving this inside the python presence condition was better.

Reply:
Actually I can try to check for the emptiness in jinja, will try that

Reply:
Ok, reverted and changed, seems fine, have a look